### PR TITLE
use a more open license

### PR DESCRIPTION
**Pull request by Helfull**

### CODE REVIEW

File: LICENSE.MD [hunk 1]
```diff
@@ -1,21 +1,13 @@
-The MIT License (MIT)
+DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE 
+                    Version 2, December 2004 
 
-Copyright (c) 2015 Everyone
+ Copyright (C) 2004 Sam Hocevar <sam@hocevar.net>
```
Comment (svenluijten) [inline LICENSE.MD:4]: Who is Sam?

Comment (cwhite92) [inline LICENSE.MD:4]: > Samuel “Sam” Hocevar is a French software and video game developer. Hocevar was the Debian project leader from 17 April 2007 to 16 April 2008.

Comment (svenluijten) [inline LICENSE.MD:4]: Thanks @cwhite92.

**Comment by phroggyy** [comment]
I think we need a discussion if this is really the way to go. Opinions @cwhite92 and @svenluijten?

**Comment by phroggyy** [comment]
@Helfull could you merge the master first and then I'll merge this

**Comment by cwhite92** [comment]
I totally missed the call to discussion you made 26 days ago. I support this license as it reflects how I choose to live my life. :shipit: :rocket:

**Comment by Helfull** [comment]
should we change the copyright owner to `everyone` or let it be `Sam`

**Comment by Helfull** [comment]
Closing, sorry for the notification, i am cleaning my PR list

**Pull request closed without merging**